sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -794,8 +794,10 @@ case class ShowCreateTableCommand(table: TableIdentifier) extends RunnableComman
   private def showHiveTableProperties(metadata: CatalogTable, builder: StringBuilder): Unit = {
     if (metadata.properties.nonEmpty) {
       val filteredProps = metadata.properties.filterNot {
-        // Skips "EXTERNAL" property for external tables
-        case (key, _) => key == "EXTERNAL" && metadata.tableType == EXTERNAL
+        // Skips all the stats info (See the JIRA: HIVE-13792)
+        case (key, _) =>
+          key == "numFiles" || key == "numRows" || key == "totalSize" || key == "numPartitions" ||
```

Comment:
Is it possible that the list of hidden properties will grow in future? If so, can we not add them with `||` here? A separated list like the `excludedTableProperties` below seems good. And we can check if the key is in the list.